README.md

```diff
@@ -693,13 +694,13 @@ $ docker run -d -it -p 7200:8000 -p 7201:8001 -p 7202:8002 \
 
 When you complete these steps, you will have multiple containers; one on each VM, and all connected to each other on the 'ml-cluster-network' network. All the containers will be part of same cluster.
 
-## How to join TLS enabled cluster
+## How to join HTTPS(TLS) enabled cluster
```

Comment:
How to join a TLS(HTTPS) enabled cluster

Reply:
Changes added.